mcodingbot/plugins/info.py

```diff
@@ -20,7 +20,7 @@ async def links(ctx: Context) -> None:
         title="Links",
         color=CONFIG.theme,
         description=(
-            f"**[mCoding site]({CONFIG.mcoding_site})**\n"
+            f"**[mCoding website]({CONFIG.mcoding_website})**\n"
             f"**[mCoding Youtube]({CONFIG.mcoding_youtube})**\n"
```

Comment:
```suggestion
            f"**[mCoding YouTube]({CONFIG.mcoding_youtube})**\n"
```
i guess it's not out of scope to suggest that

Reply:
agreed